### DISCUSSION TITLE
Localisation support

### DISCUSSION THREAD

**Discussion by ilarimikkonen** [magda-io/magda#3502] [General]
Hi,
is UI available in any other language except english?
Ilari

**Comment by t83714** [comment]
We use [react-i18next](https://www.npmjs.com/package/react-i18next) to manage languages.

You can add language text items for different languages via [Content API](https://dev.magda.io/api/v0/apidocs/index.html#api-Content-PutV0ContentContentid) or directly edit content DB.

We will also add a UI for it soon.

Most but not all text label are covered by language items in the DB -- we will also increase the coverage of it in future.
